bVNC/src/main/java/com/iiordanov/bVNC/dialogs/RateOrShareFragment.kt

```diff
@@ -60,10 +62,53 @@ class RateOrShareFragment : DialogFragment() {
         }
         versionAndCode = v.findViewById<View>(R.id.versionAndCode) as TextView
         versionAndCode?.text = Utils.getVersionAndCode(v.context)
-
+        setVisibilityOfElements(v)
         return v
     }
 
+    fun setTitle() {
+        if (Utils.isCustom(context)){
+            try {
+                dialog?.setTitle(
+                    getString(
+                        requireContext().resources.getIdentifier(
+                            Utils.getStringConfigAttribute(App.configFileReader.configData, TAG.replaceFirstChar { it.lowercase() }, "title", "key"),
+                            "string",
+                            Utils.pName(context)
+                        )
+                    )
+                )
+                return
+            }
+            catch (e: Exception) {
```

Comment:
Instead of catching an Exception and then re-throwing "if it's an NPE" later, why not just catch only NullPointerException and let any other kind of exception get automatically not caught and thrown?